File: CH3-desicion-sructure-and-boolean-logic/EX1-roman-numerals/source-code/romanNumerals.java
import java.util.Scanner;

public class romanNumerals {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int number;

        System.out.print("enter a number range 1 through 10 :");
        number = input.nextInt();

        switch (number) {
            case 1:
                System.out.print("I");
                break;
            case 2:
                System.out.print("II");
                break;
            case 3:
                System.out.print("III");
                break;
            case 4:
                System.out.print("IV");
                break;
            case 5:
                System.out.print("V");
                break;
            case 6:
                System.out.print("VI");
                break;
            case 7:
                System.out.print("VII");
                break;
            case 8:
                System.out.print("VIII");
                break;
            case 9:
                System.out.print("IX");
                break;
            case 10:
                System.out.print("X");
                break;
            default:
                System.out.print("Input not valid !!! enter a number range 1 through 10");
                break;
        }

        input.close();
    }
}

File: CH3-desicion-sructure-and-boolean-logic/EX1-roman-numerals/source-code/romanNumerals.js
const input = require("prompt-sync")();

number = Number(input("enter a number range 1 through 10 :"));

switch (number) {
  case 1:
    console.log("I");
    break;
  case 2:
    console.log("II");
    break;
  case 3:
    console.log("III");
    break;
  case 4:
    console.log("IV");
    break;
  case 5:
    console.log("V");
    break;
  case 6:
    console.log("VI");
    break;
  case 7:
    console.log("VII");
    break;
  case 8:
    console.log("VIII");
    break;
  case 9:
    console.log("IX");
    break;
  case 10:
    console.log("X");
    break;
  default:
    console.log("Input not valid !!! enter a number range 1 through 10");
    break;
}
